snuba/api.py

```diff
@@ -311,7 +311,8 @@ def parse_and_run_query(validated_body, timer):
         having_clause,
         order_clause,
         limitby_clause,
-        limit_clause
+        limit_clause,
+        'FORMAT JSON'
```

Comment:
This `FORMAT` parameter seems to be disregarded by the native client, since it's just going to return the native format anyway.